example/pythonlib/linting/3-coverage/build.mill

```diff
@@ -0,0 +1,44 @@
+// Mill's support for code coverage analysis is implemented by
+// https://coverage.readthedocs.io/[the coverage.py package].
+//
+// You can use it by extending `CoverageTests` in your test module.
+
+import mill._, pythonlib._
+
+object `package` extends RootModule with PythonModule {
```

Comment:
Shouldn't this need to inherit from `CoverageModule` in order for `CoverageTests` to be available?

Reply:
Ah I misread it, `CoverageTests` is a sibling trait, not a nested one